pkg/crdpuller/discovery.go

```diff
@@ -142,6 +142,8 @@ func (sp *schemaPuller) PullCRDs(context context.Context, resourceNames ...strin
 				continue
 			}
 
+			resourcesToPull.Delete(groupResource.String(), apiResource.Name)
```

Comment:
Could you please add a comment why we're doing this here? Thanks!

Reply:
I added a comment, yes 👍